rpc/rpc.go

```diff
@@ -663,6 +663,16 @@ func (c *Conn) handleBootstrap(ctx context.Context, id answerID) error {
 	return err
 }
 
+func makeIdempotent(f func()) func() {
```

Comment:
This is a minor issue of taste/style, but I find that `idempotent(foo)` somehow reads better.   I recognize it runs somewhat contrary to the verb-y function names commonly used in Go, so it's your call.

Reply:
Done.